iceoryx_hoofs/source/posix_wrapper/file_lock.cpp

```diff
@@ -217,7 +219,7 @@ FileLockError FileLock::convertErrnoToFileLockError(const int32_t errnum, const
     }
     case ENOENT:
     {
-        LogError() << "directory \"" << platform::IOX_LOCK_FILE_PATH_PREFIX << "\""
+        LogError() << "directory \"" << &platform::IOX_LOCK_FILE_PATH_PREFIX[0] << "\""
```

Comment:
This looks quite ugly and is even error prone if one uses something different than `0`. I'd rather find a better solution in the mid to long run like `iox::a2p(platform::IOX_LOCK_FILE_PATH_PREFIX)`, just better.